libtenzir/builtins/operators/python.cpp

```diff
@@ -147,6 +147,10 @@ class python_operator final : public crtp_operator<python_operator> {
   auto execute(generator<table_slice> input, operator_control_plane& ctrl) const
     -> generator<table_slice> {
     try {
+      // Signal successful startup immediately to avoid 504 timeout errors in
+      // the platform instead of waiting until the operator actually
+      // initialized.
```

Comment:
Suggestion: "to avoid blocking the `/launch` endpoint" / "to avoid blocking the operator".

The 504 error is just a secondary effect, ie. Jannis originally complained due to the effect on interactive CLI usage.